docs/operations/images.md

```diff
@@ -123,6 +123,12 @@ az vm image list --all --output table \
   --publisher Debian --offer debian-11 --sku 11-gen2
 ```
 
+### Debian 12 (Bookworm)
+
+Debian 12 is based on Kernel version **6.1** which has no known major Kernel bugs and fully supports all Cilium features.
+
+
```

Comment:
Please add a link to this in the Matrix.
```suggestion
At the moment there is no official image published.
```

Reply:
What version should I put as the starting version? `1.27`? `1.26.3`?

Reply:
I would go with 1.26.3.